app/helpers/sessions_helper.rb

```diff
@@ -15,4 +15,8 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  def admin?
+    current_user.role == Settings.user.admin
```

Comment:
tìm hiểu lại `enum in rails` đi, ko cần so sánh thế này, enum có hỗ trợ check cái này đấy